### PR TITLE
Doing a pass to make sure that DevTools is correctly used

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/10202

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by atsansone** [comment]
Looks good, but may want to check with @mit about naming. I think I remember being told to just use "Dart DevTools" without Flutter in the name at all.

**Comment by sfshaza2** [comment]
@mit-mit, the last time we discussed this, we were told to call it either Flutter DevTools or Dart DevTools, or both. @atsansone says that the advice is now to call them Dart DevTools. Please clarify.

File: src/content/tools/devtools/app-size.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ the Wonderous App using DevTools, check out the
 [App Size Tool tutorial][app-size-tutorial]. Various strategies
 to reduce an app's size are also discussed.
 
-[Use the treemap]: #using-the-treemap
+[Use the treemap]: #use-the-treemap
```
Comment (domesticmouse) [inline src/content/tools/devtools/app-size.md:270]: Won't this change break incoming references?

Comment (sfshaza2) [reply src/content/tools/devtools/app-size.md:270]: If there are any, yup.

**Comment by sfshaza2** [comment]
@mit-mit, I'm going to land this. But, when you return, let me know if you want to chat about it.

**Pull request merged**